meta-resin-common/recipes-support/resin-device-register/resin-device-register/resin-device-register

```diff
@@ -62,7 +57,7 @@ while true; do
         if [ "$status_code" -eq 201 ]; then
             device_id=$(jq -r '.id' /var/log/register-device-curl.log)
             config_json=`cat $CONFIG_PATH`
-            echo $config_json | jq ".uuid=\"$uuid\"|.registered_at=\"`date +%s`\"|.deviceId=\"$device_id\"" > $CONFIG_PATH
+            echo $config_json | jq ".registered_at=\"`date +%s`\"|.deviceId=\"$device_id\"" > $CONFIG_PATH
```

Comment:
Hm ok. You need the value later in registration indeed. And this value comes now from a dependent service. Looks good to me.